### ISSUE TITLE
U+1B34 BALINESE SIGN REREKAN blocks contextual U+1B1B BALINESE LETTER JA JERA

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1506]
### Font
NotoSerifBalinese-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/af306de71ce51b9ce75389b163026427953606cf/phaseIII_only/unhinted/ttf/NotoSerifBalinese/NotoSerifBalinese-Regular.ttf
Date: 2019-04-25

### Font version
Version 2.000

### Issue
U+1B34 BALINESE SIGN REREKAN blocks the shorter form of <i>jha</i> before a subjoined letter.

### Character data
ᬛ᭄ᬓᬛ᬴᭄ᬓ
U+1B1B BALINESE LETTER JA JERA
U+1B44 BALINESE ADEG ADEG
U+1B13 BALINESE LETTER KA
U+1B1B BALINESE LETTER JA JERA
U+1B34 BALINESE SIGN REREKAN
U+1B44 BALINESE ADEG ADEG
U+1B13 BALINESE LETTER KA

### Screenshot
![ᬛ᭄ᬓᬛ᬴᭄ᬓ](https://user-images.githubusercontent.com/1124347/57236525-452fb000-6ff3-11e9-8718-8450afabbebb.png)

**Comment by NorbertLindenberg** [comment]
According to [Proposal for encoding the Balinese script in the UCS](https://www.unicode.org/L2/L2005/05008-n2908-balinese.pdf), U+1B1B only occurs in a single word: ᬦᬶᬃᬛᬭ nirjhara ‘pond’.

**Comment by kamholz** [comment]
@NorbertLindenberg: that may be correct for Balinese, but Balinese script can also be used to write Kawi and Sanskrit, which would use this letter in more contexts. I don't know how normal it is for a conjunct to appear below, however.

**Comment by kamholz** [comment]
Addendum to previous comment after discussion with @adtbayuperdana. JA JERA is probably not more common in Kawi than Balinese, and it's unclear that actual Sanskrit texts exist in Balinese script (probably what is meant is larger presence of Sanskrit vocabulary). That being said, JA JERA occurs in more than just the word ᬦᬶᬃᬛᬭ nirjhara ‘pond’. Brandes's edition of Nagarakrtagama (chapter 2 verse 2 line 3) contains JA JERA in the word Majapahit:

![Nagarakrtagama](https://user-images.githubusercontent.com/1434631/59173261-f0ef8280-8b00-11e9-9f8d-96769d7daad1.jpeg)

He also found some examples of JA JERA in [this lontar](https://archive.org/details/uger-uger-pasang-aksara-bali), which seems to be a spelling exercise.

The proposal is therefore incomplete regarding JA JERA. However, @adtbayuperdana believes that it is extremely unlikely if not impossible for JA JERA to co-occur with REREKAN. REREKAN is used to write sounds that Balinese script otherwise can't represent, and he doesn't know of any combination with JA JERA. Presumably if someone wanted to modify the JA sound they would just use JA.

**Comment by dscorbett** [comment]
Fixing this is as simple as specifying a mark class in GSUB lookup <span>#</span>4. Even if this cluster is unattested, the font’s current behavior is clearly inconsistent with how the cluster would look.

**Comment by kamholz** [comment]
@dscorbett: fair enough. I mean, even the uni1B1B.alt glyph (shorter JA JERA) is unattested, as far as I know. :-)
